### DISCUSSION TITLE
Official announcement by Joomlaworks says K2 will not support J4/5. Time to disscuss the future of K2ForJ4?

### DISCUSSION THREAD

**Discussion by krmrtrzv** [eworkers/K2ForJ4#88] [Ideas]
Hello @eworkers, 

On May 30, 2024 Joomlaworks made an announcement on their blog about the change of their plans: "K2 will not be made available for Joomla 4/5 - change of course" <https://www.joomlaworks.net/blog/item/314-k2-will-not-be-made-available-for-joomla-4-5-change-of-course>

As I am interested in keeping all my websites using K2 in long-term perspective, and I believe we all are, I wish to bring the disscussion here what are the prospects of turning K2ForJ4 into official extension, so all K2 specific extensions could be brought back to life and the horizon moves from temporary fix to stable plan.

Thank you for the initiative, for all your efforts and your thoughts on this!

**Comment by litho-publisher** [comment]
Hi
I'm in!!!!
I would also be happy if there would be the possibility to install more plugins like:
https://kubik-rubik.de/de/sige-simple-image-gallery-extended
As a replacement for the old gallery. Only the frontend upload is missing.
Thanks!!!!
There is no better editing system for end customers!
